### DISCUSSION TITLE
Centaurus Open Source Community Meeting

### DISCUSSION THREAD

**Discussion by zmn223** [CentaurusInfra/arktos#1422] [Open Source Meeting]
### **Centaurus Open Source Community Meeting - Container Engine** 

**Monday 6:00 PM Pacific Time, Weekly**
Zoom: https://futurewei.zoom.us/j/92636035970

[**Meeting Agenda**](https://docs.google.com/document/d/1Vebq1qknCc0c6SIXdMN9pUeqVw0Mph1OKLlRJNa3cyg/edit#) 

**5/10:** Arion Introduction [Arion_control_plane_design.pdf](https://github.com/CentaurusInfra/arktos/files/8655942/Arion_control_plane_design.pdf)
**5/11:** Centaurus Open Source Project Overview  [Centaurus Open Source Projects.pptx](https://github.com/CentaurusInfra/arktos/files/8664811/CloudLab-OSProjects.pptx)
**5/23** Quark Weekly Update 
**6/6** Weekly Update
**6/13** Open source collaboration and operation discussion
**6/20** Weekly sync
**6/27** Weekly sync
**7/4** Holiday schedule, meeting canceled
**7/11** No agenda item, meeting dismissed
**7/18** Recent work sync up
**7/25** Weekly sync
**8/1** Weekly sync
**8/8** Meeting dismissed, as no agenda item proposed. 
**8/15** Weekly Sync
**8/22** Weekly Sync
**8/29** Weekly sync and open source operation update
**9/5** Weekly Sync
**9/12** 

- RDMA Network Discussion
   - RDMA Benchmark Sharing https://github.com/QuarkContainer/Quark/blob/main/doc/RDMA%20Benchmark%20(version%201).xlsx
   - TCP Socket over RDMA [TCP Socket over RDMA (2).pptx](https://github.com/CentaurusInfra/arktos/files/9550579/TCP.Socket.over.RDMA.2.pptx)
   - TCP Socket over RDMA (TSoR) Integration with K8S https://docs.google.com/presentation/d/1UcMsW45nNzapWabyhEJ5Ve0hxQ0yYgd0eRtlCydILl8/edit#slide=id.p
- Quark Hibernate Container  https://docs.google.com/presentation/d/1qdkgw3qUnig-MLrPGjCw5t1SO4M9IXy65FM1iP4VRLc/edit#slide=id.p
- Kubemark for Network Performance 
[KubemarkNetPerf.pdf](https://github.com/CentaurusInfra/arktos/files/9610562/KubemarkNetPerf.pdf)
https://docs.google.com/document/d/1cx6Yyz7TyAwDjQav9Bq1dbBqbb9Gj8aDwwj5HxcSf8g/edit#heading=h.43a9jcv1u3mu

**9/19** Weekly Sync
**9/26** Weekly Sync + KubeMark for Network Performance Demo 
- Kubemark for Network Performance 
[KubemarkNetPerf.pdf](https://github.com/CentaurusInfra/arktos/files/9610562/KubemarkNetPerf.pdf)
https://docs.google.com/document/d/1cx6Yyz7TyAwDjQav9Bq1dbBqbb9Gj8aDwwj5HxcSf8g/edit#heading=h.43a9jcv1u3mu
Latest doc: [KubemarkCloudNetworkSim.pdf](https://drive.google.com/file/d/11CY5ut0bD6CEghQmgSZPYQuR3yv8cw_O/view?usp=sharing)

**9/29** Ad-hoc : Kuasar open source operation discussion
**10/3** Meeting canceled.
**10/10** Weekly Sync
**10/12** Ad-hoc: Open source operation discussion 
**10/17** Weekly Sync
**10/23** Container Network Discussion (Ad-hoc) 
**10/24** Weekly Sync
**10/31** Meeting canceled. 
**11/07** Weekly Sync
**11/14** Weekly Sync
**11/21** KubeCon report sharing [KubeConNA2022Report 1.pdf] (https://github.com/CentaurusInfra/arktos/files/10045740/KubeConNA2022Report.1.pdf)
**11/28** Weekly Sync
**11/30** Ad-hoc meeting - [Quark Introduct.pptx](https://github.com/CentaurusInfra/arktos/files/10126118/Quark.Introduct.pptx)
**12/5** Meeting Canceled
**12/12** Meeting Canceled
**12/19** Weekly Sync

[**Meeting Notes**](https://docs.google.com/document/d/1Vebq1qknCc0c6SIXdMN9pUeqVw0Mph1OKLlRJNa3cyg/edit#)

### **Centaurus Open Source Community Meeting**
**Tuesday 6:00 PM Pacific Time, Weekly**
Zoom: https://futurewei.zoom.us/j/91236073812

[**Meeting Agenda**](https://docs.google.com/document/d/1Cwpp44pQhMZ_MQ4ebralDHCt0AZHqhSkj14kNAzA7lY/edit#) 

**5/16** Introduction to Fornax-serverless  [Edge Serverless Computing.pptx](https://github.com/CentaurusInfra/arktos/files/8691538/Edge.Serverless.Computing.pptx)
**5/17** [Regionless KV Store ](https://github.com/CentaurusInfra/arktos/files/8712260/open-source-meeting-05-17.pptx) update and remaining questions from Hadean OS, Gamebus Google, and Spacial OS discussion
**5/24** Fornax Serverless Platform Discussion and related design sharing [Fornax Serverless compare to VM platform.pptx](https://github.com/CentaurusInfra/arktos/files/8766865/Fornax.Serverless.compare.to.VM.platform.pptx)
**5/31** Fornax Scheduling Discussion [fornax scheduling design.pptx](https://github.com/CentaurusInfra/arktos/files/8809624/fornax.scheduling.design.pptx)
**6/7** Quark overview [Quark.pptx](https://github.com/CentaurusInfra/arktos/files/8856114/Quark.pptx)
**6/14** Centaurus Weekly Sync
**6/21** [Regionless KV Store: consistency discussion and demo](https://github.com/CentaurusInfra/arktos/files/8953533/open-source-meeting-06-20.pdf)
**6/28** Community member to share their problems and visions on auto scaling 
**7/12** Centaurus Weekly Sync
**7/19** Centaurus Weekly Sync
**7/26** Centaurus Weekly Sync + Knowledge sharing on 3D rendering/neural rendering [Neural Rendering.pptx](https://github.com/CentaurusInfra/arktos/files/9192486/Neural.Rendering.pptx)
**8/2** Centaurus Weekly Sync + game use case/demo sharing 
**8/9** Centaurus project status updates + CDN use case sharing from the community
**8/16** Meeting dismissed due to members' availability. 
**8/23** Meeting dismissed as no agenda items proposed.
**8/30** Project update: 
- Regionless Storage Service [830_community_meeting.pptx](https://github.com/CentaurusInfra/arktos/files/9457659/830_community_meeting.pptx)
- Global Resource Service https://github.com/CentaurusInfra/global-resource-service/blob/main/docs/GRS930-Progress-2022-0830.pdf
- Quark [TCP Socket over RDMA.pptx](https://github.com/CentaurusInfra/arktos/files/9457671/TCP.Socket.over.RDMA.pptx)
- Fornax Serverless https://github.com/CentaurusInfra/fornax-serverless

**9/6** Meeting dismissed as no agenda items proposed.
**9/13** Quark Hibernate Container Sharing https://docs.google.com/presentation/d/1qdkgw3qUnig-MLrPGjCw5t1SO4M9IXy65FM1iP4VRLc/edit#slide=id.p 
**9/20** (Meeting canceled)
**9/27** Weekly Sync
**10/4** Meeting canceled
**10/11** 930 Release Sharing (Part 1) 

- Global Resource Service 930 release: https://github.com/CentaurusInfra/global-resource-service/releases/tag/v0.2.0-alpha

**10/18** 930 Release Sharing (Part 2) [10-18-22.Community.Meeting.pptx](https://github.com/CentaurusInfra/arktos/files/9815947/10-18-22.Community.Meeting.pptx)
**10/25** Fornax Serverless Platform Latest Update - https://github.com/CentaurusInfra/fornax-serverless/discussions/113 

- Hibernate Container https://docs.google.com/presentation/d/1qdkgw3qUnig-MLrPGjCw5t1SO4M9IXy65FM1iP4VRLc/edit#slide=id.p
- Serverless Platform [Serverless Platform-Updated.pptx](https://github.com/CentaurusInfra/arktos/files/9863933/Serverless.Platform-Updated.pptx)

**11/1** Global Scheduler Requirement Discussion https://github.com/CentaurusInfra/arktos/discussions/1425

**11/8** Learning and Sharing Session 

- Actor Model and Dapr [ActorModelAndDapr.pptx](https://github.com/CentaurusInfra/arktos/files/9965435/ActorModelAndDapr.pptx)
- Akka [Akka-Intro.pptx](https://github.com/CentaurusInfra/arktos/files/9965443/Akka-Intro.pptx)

**11/15** Meeting canceled
**11/22**  Centaurus Weekly Sync
**11/29** Centaurus Weekly Sync
**12/6** Massive Online Player use case and requirement discussion [Centaurus Open Source Community Meeting Agenda 12062022.pptx](https://github.com/CentaurusInfra/arktos/files/10144982/Centaurus.Open.Source.Community.Meeting.Agenda.12062022.pptx)
[community_meeting_dec0622.pptx](https://github.com/CentaurusInfra/arktos/files/10171837/community_meeting_dec0622.pptx)
**12/13** Meeting Canceled
**12/20** Meeting Canceled
**12/27** Quick discussion on  VM auto scaling usage of durable function https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview?tabs=csharp

[**Meeting Notes**](https://docs.google.com/document/d/1Cwpp44pQhMZ_MQ4ebralDHCt0AZHqhSkj14kNAzA7lY/edit#)

**Comment by pdgetrf** [comment]
[10-18-22 Community Meeting.pptx](https://github.com/CentaurusInfra/arktos/files/9815924/10-18-22.Community.Meeting.pptx)
